src/viam/services/navigation/navigation.py

```diff
@@ -5,6 +5,7 @@
 
 from ..service_base import ServiceBase
 from . import GeoObstacle, GeoPoint, Mode, Waypoint
+from viam.proto.service.navigation import Path
```

Comment:
@njooma here and elsewhere we were just importing `Path` from `viam.proto.service.navigation`. I see some other API types being imported from `.`; how does that work? We added `Path` (along with `GetPathsRequest` and `GetPathsResponse`) to `src/viam/proto/service/navigation/__init__.py`, but I don't think we were able to find `Path` in `.` here.

Reply:
The `.` syntax is only if the class is included in the `src/viam/services/navigation/__init__.py`, not the `.../proto/...` file.